### PR TITLE
statistics: remove useless field in jsonColumn

### PR DESCRIPTION
**Pull request by fipped**
`AlreadyLoad` is used to marked whether the histograms stats are in the cache or not.
Since the stats will be loaded into the storage directly, the `AlreadyLoad` is useless now. We can remove it.
@lamxTyler @hanfei1991 PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
Nice catch, cleaning up makes me feels good.

File: statistics/dump.go [hunk 1]
```diff
@@ -35,21 +35,24 @@ type JSONTable struct {
...
 	NullCount         int64           `json:"null_count"`
 	LastUpdateVersion uint64          `json:"last_update_version"`
 }
 
-type jsonIndex struct {
-	Histogram         *tipb.Histogram `json:"histogram"`
-	CMSketch          *tipb.CMSketch  `json:"cm_sketch"`
-	NullCount         int64           `json:"null_count"`
-	LastUpdateVersion uint64          `json:"last_update_version"`
+func dumpJSONCol(hist *Histogram, CMSketch *CMSketch, NullCount int64, LastUpdateVersion uint64) *jsonColumn {
```
Comment (lamxTyler) [inline statistics/dump.go]: Do not need to pass the `NullCount` and `LastUpdateVersion`.

File: statistics/dump.go [hunk 2]
```diff
@@ -58,44 +61,31 @@ func (h *Handle) DumpStatsToJSON(dbName string, tableInfo *model.TableInfo) (*JS
...
-		}
-		jsonTbl.Indices[idx.Info.Name.L] = jsonIdx
+		jsonTbl.Indices[idx.Info.Name.L] = dumpJSONCol(&idx.Histogram, idx.CMSketch)
 	}
 	return jsonTbl, nil
 }
 
+func loadHist(hist *Histogram, ID, NullCount int64, LastUpdateVersion uint64) Histogram {
+	hist.ID, hist.NullCount, hist.LastUpdateVersion = ID, NullCount, LastUpdateVersion
+	hist.PreCalculateScalar()
```
Comment (lamxTyler) [inline statistics/dump.go]: This is not necessary? Since we just write it to the storage.

**Pull request merged**